src/code42cli/cmds/securitydata.py

```diff
@@ -206,6 +209,9 @@ def search(
         extractor.extract(*state.search_filters)
     if handlers.TOTAL_EVENTS == 0 and not errors.ERRORED:
         echo("No results found.")
+    else:
+        output = _process_events(format, include_all, handlers.EVENTS)
+        click.echo_via_pager(output)
```

Comment:
how does `echo_via_pager` behave when the command is run in the background?

Reply:
I'm pretty sure it detects if you're in a TTY and if so, attempts to pipe to your default pager, otherwise does the same as `echo()` does. So if you're piping the output to another process or file, it skips the pager.

Reply:
Confirmed it skips the pager.